File: frontend/src/constants/GroundTaskStatus.ts
export const GroundTaskStatus = ["PENDING", "IN_PROGRESS", "BLOCKED", "COMPLETED"] as const;
export type GroundTaskStatus = (typeof GroundTaskStatus)[number];
export const GroundTaskStatusText: Record<GroundTaskStatus, string> = {
  PENDING: "待执行",
  IN_PROGRESS: "执行中",
  BLOCKED: "阻塞",
  COMPLETED: "已完成"
};

File: backend/src/constants/GroundTaskStatus.go
package constants

// GroundTaskStatus enumerates the lifecycle of a ground task.
// A task is considered finished only when it equals COMPLETED.
var GroundTaskStatus = []string{"PENDING", "IN_PROGRESS", "BLOCKED", "COMPLETED"}

const (
	GroundTaskStatusPending    = "PENDING"
	GroundTaskStatusInProgress = "IN_PROGRESS"
	GroundTaskStatusBlocked    = "BLOCKED"
	GroundTaskStatusCompleted  = "COMPLETED"
)
